addon/helpers/percentage.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const {
+  Helper
+} = Ember;
+
+export function percentage(input) {
+  return Math.round((input[0] / input[1]) * 100);
```

Comment:
maybe be carrefull if the `input[1]` is null / 0. An early return can do the job